README.md

```diff
@@ -75,12 +80,6 @@ To build SciLexer.dll without PCRE support (for both 32-bit & 64-bit):
     - `nmake NOBOOST=1 -f scintilla.mak`         (normal build)
     - `nmake NOBOOST=1 DEBUG=1 -f scintilla.mak` (debugging build)
 
- 4. Copy `SciLexer.dll` from `scintilla\bin\` to the same directory as `notepad++.exe`.
-    - For the `Unicode Release` configuration, the output directory
-      (where `notepad++.exe` is) is `PowerEditor\bin\`.
-    - For the `Unicode Debug` configuration, the output directory
-      (where `notepad++.exe` is) is `PowerEditor\visual.net\Unicode Debug\`.
-

```

Comment:
This copy instruction is required for both the Scintilla build _with_ and _without_ Boost.